scripts/backup/backup

```diff
@@ -3,9 +3,10 @@
 set -euo pipefail
 
 UMBREL_ROOT="$(readlink -f $(dirname "${BASH_SOURCE[0]}")/../..)"
-BACKUP_FOLDER="backup"
-BACKUP_ROOT="${UMBREL_ROOT}/${BACKUP_FOLDER}"
-BACKUP_FILE="${UMBREL_ROOT}/backup.tar.gz.pgp"
+BACKUP_ROOT="${UMBREL_ROOT}/.backup/$RANDOM"
```

Comment:
Wow, didn't know there was an always available `$RANDOM` in shell 😍

Reply:
It's not very random, in fact it's not even uniformly random, but yeah, it's always available in a shell and fine for stuff like this.

It's also not actually a variable but a function, so you can use it multiple times in the same line

e.g:

```
$ echo "${RANDOM}:${RANDOM}:${RANDOM}"
16448:13088:7550
```